app/views/notifications/_article.html.erb

```diff
@@ -3,13 +3,29 @@
   <%= render "notifications/shared/profile_pic", json_data: json_data %>
 
   <div class="notification__content">
-    <header class="mb-4">
-      <h2 class="fs-base fw-normal">
-        <%= message_user_acted_maybe_org(json_data, "views.notifications.new_post.verb_html", if_org: "views.notifications.new_post.if_org_html") %>
-      </h2>
-      <p class="lh-tight"><small class="fs-s color-base-60"><%= time_ago_in_words json_data["article"]["published_at"], scope: :"datetime.distance_in_words_ago" %></small></p>
-    </header>
-
-    <%= render "notifications/shared/article_preview", json_data: json_data, notification: notification, context: "default" %>
+    <% if notification.action == "Published" %>
+      <header class="mb-4">
+        <h2 class="fs-base fw-normal">
+          <%= message_user_acted_maybe_org(json_data, "views.notifications.new_post.verb_html", if_org: "views.notifications.new_post.if_org_html") %>
+        </h2>
+        <p class="lh-tight"><small class="fs-s color-base-60"><%= time_ago_in_words json_data["article"]["published_at"], scope: :"datetime.distance_in_words_ago" %></small></p>
+      </header>
+
+      <%= render "notifications/shared/article_preview", json_data: json_data, notification: notification, context: "default" %>
+    <% elsif notification.action == "Moderation" %>
```

Comment:
(question): Will the `notification.action` be set to `"Moderation"` for only for the welcome email?

Reply:
Yes, currently the only moderator notification on an article is when it's the user's first or second published article. The method that does that check is new to this PR, and it's the only one attaching the "Moderation" value in the notification's action key, so only the welcome should be hitting this part of the view.